<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI</class>
 <widget class="QMainWindow" name="UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Human Counter</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lb_show">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="title">
          <string>Source</string>
         </property>
         <widget class="QRadioButton" name="radioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>From file:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_video">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>161</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_Browse">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>90</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="title">
          <string>Controller</string>
         </property>
         <widget class="QPushButton" name="btn_spause">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>91</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_stop">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>30</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Stoped</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>220</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>220</height>
          </size>
         </property>
         <property name="title">
          <string>Processing</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Counter:</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="txt_sum">
          <property name="geometry">
           <rect>
            <x>43</x>
            <y>50</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLCDNumber" name="txt_men">
          <property name="geometry">
           <rect>
            <x>43</x>
            <y>110</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Men:</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="txt_women">
          <property name="geometry">
           <rect>
            <x>43</x>
            <y>170</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Women:</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Information</string>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Fps:</string>
          </property>
         </widget>
         <widget class="QLabel" name="lb_pfs">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>30</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
